SwanContents/swancontents/filemanager/swan_eos_filemanager.py

```diff
@@ -9,7 +9,7 @@
 import shutil
 
 
-class SwanEosFileManager(ProjectsMixin, SwanFileManagerMixin, LargeFileManager):
+class SwanEosFileManager(ProjectsMixin, SwanFileManagerMixin, AsyncLargeFileManager):
```

Comment:
Does this change in the superclass have any implication? There's no change in the private methods we define in this class, but perhaps this class is used from somewhere else and we are invoking the coroutine equivalent of methods that we used to invoke on it before?

Reply:
Yes we are invoking the coroutine equivalent of methods that we used to invoke (https://github.com/jupyter-server/jupyter_server/blob/c67a46befdcc972436150e9876b5d25d3c33ea56/jupyter_server/services/contents/largefilemanager.py#L85)